### PR TITLE
Merge `fast` odh-manifest changes of ODH ModelMesh-Serving into night…

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
…lies of RHODS odh-manifest

Jira : https://issues.redhat.com/browse/RHODS-9778

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [ ] JIRA link(s):
- [ ] The Jira story is acked
- [ ] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

### CODE REVIEW

File: model-mesh/base/params.env [hunk 1]
```diff
@@ -1,7 +1,7 @@
 monitoring-namespace=opendatahub
-odh-mm-rest-proxy=quay.io/opendatahub/rest-proxy:v0.10.0
-odh-modelmesh-runtime-adapter=quay.io/opendatahub/modelmesh-runtime-adapter:v0.11.0-alpha
-odh-modelmesh=quay.io/opendatahub/modelmesh:v0.11.0-alpha
-odh-openvino=quay.io/opendatahub/openvino_model_server:2022.3-gpu
-odh-modelmesh-controller=quay.io/opendatahub/modelmesh-controller:v0.11.0-alpha
-odh-model-controller=quay.io/opendatahub/odh-model-controller:v0.11.0-alpha
+odh-mm-rest-proxy=quay.io/opendatahub/rest-proxy:v0.11.0-rc0
```
Comment (VedantMahabaleshwarkar) [inline model-mesh/base/params.env:2]: these should reference the fast tag images right? @Jooho

**Review by VedantMahabaleshwarkar — CHANGES_REQUESTED** [review]

File: model-mesh/base/params.env (hunk 1 above)
Comment (Jooho) [reply model-mesh/base/params.env:2]: Yes

Comment (vaibhavjainwiz) [reply model-mesh/base/params.env]: done

**Review by VedantMahabaleshwarkar — APPROVED** [review]
/lgtm

**Pull request merged**